### DISCUSSION TITLE
How to run qpbenchmark step-by-step?

### DISCUSSION THREAD

**Discussion by ottapav** [qpsolvers/qpbenchmark#105] [General]
Before the repository was renamed and test sets reorganized, I had a script that ran the qpsolvers_benchmark in the following steps
1) clone qpsolvers_benchmark repository
2) activate a virtual environment
2) pip install qpsolvers_benchmark, qpsolvers, and other packages
3) benchmark.py maros_meszaros_dense_posdef run

Currently, I'm struggling with where the test sets should be located - I'm getting ModuleNotFoundError. @stephane-caron Could you please write what are the steps to run qpbenchmark now?

**Comment by stephane-caron** [comment]
Sure. Those are close to what you were already doing:

- Same steps as 1-3, with qpbenchmark rather than qpsolvers_benchmark
- After pip installing qpbenchmark, you will get a `qpbenchmark` binary installed
- For step 4, the test set has moved to [maros_meszaros_qpbenchmark](https://github.com/qpsolvers/maros_meszaros_qpbenchmark#usage) as test sets are more independent now, so you will now want to:

```
git clone https://github.com/qpsolvers/maros_meszaros_qpbenchmark.git
```

- Go to that repository and run the Maros-Meszaros subset as before, but calling `qpbenchmark` now:

```
qpbenchmark ./maros_meszaros_dense_posdef.py run
```

Results will be written to the results sub-directory of `maros_meszaros_qpbenchmark`.

**Reply by ottapav** [reply]
Thanks. I'm trying, yet when I run the command, I get
![image](https://github.com/qpsolvers/qpbenchmark/assets/10757098/72690e88-25b1-4ed8-9d66-63ecf550c15a)
Could you let me know if you're familiar with this issue?

**Reply by stephane-caron** [reply]
Sorry that one is on me. There were some pending updates not covered by unit tests (now covered by the [CI of qpbenchmark](https://github.com/qpsolvers/qpbenchmark/actions/workflows/ci.yml)). Can you try again with the latest [qpbenchmark v2.1.0](https://github.com/qpsolvers/qpbenchmark/releases/tag/v2.1.0)?

**Reply by ottapav** [reply]
It works perfectly now. Thanks for the super-fast action.
